ex03.ex

```diff
@@ -59,7 +59,14 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def parity(true),  do: :even
+  def parity(false), do: :odd
+
+  def odd_even([ h, h1 | t ]) do
```

Comment:
# -1

You could have done

 ~~~ elixir
[ parity(Integer.is_even(h)) | odd_even(t) ]
~~~